cmake/BLASFinder.cmake

```diff
@@ -272,6 +280,20 @@ if (test_all OR test_default)
     debug_print_list( "default" )
 endif()
 
+#---------------------------------------- Intel OneMKL
+if (test_all OR test_onemkl)
+    # IntelLLVM compiler + OpenMP: require intel_thread library.
+    if (test_int)
+        list( APPEND blas_name_list "Intel OneMKL lp64,  Intel threads (iomp5), ifort")
+        list( APPEND blas_libs_list "-lmkl_intel_lp64  -lmkl_intel_thread -lmkl_core" )
+    elseif (test_int64)
+        list( APPEND blas_name_list "Intel OneMKL ilp64, Intel threads (iomp5), ifort")
+        list( APPEND blas_libs_list "-lmkl_intel_ilp64 -lmkl_intel_thread -lmkl_core" )
+    else ()
+        message( "Skipping OneMKL for non-threaded, non-IntelLLVM compiler, upsupported" )
```

Comment:
"unsupported" typo.

Reply:
This line no longer exists, since this oneMKL search was merged with general MKL search.